content/en/docs/quickstart.md

```diff
@@ -43,86 +43,90 @@ notation list --plain-http $IMAGE
 
+The following command generates a test key and a self-signed X.509 certificate under the `$HOME/.config/notation/localkeys` directory. The test key is set as a default signing key using `--default` flag. The self-signed certificate is added to a named trust store `wabbit-networks.io` of type `ca`.
+
 ```console
 notation cert generate-test --default "wabbit-networks.io"
 ```
 
-The output of the above command shows the location of the public and private key generated. For example:
-
-```output
-$ notation cert generate-test --default "wabbit-networks.io"
-generating RSA Key with 2048 bits
-generated certificates expiring on 2022-10-01T20:18:37Z
-wrote key: <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.key
-wrote certificate: <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
-wabbit-networks.io: added to the key list
-wabbit-networks.io: marked as default
-```
-
-## Sign the image
-
-Use `notation sign` to sign the container image.
+Use `notation key list` to confirm the signing key. Key name with a `*` prefix is the default key.
 
 ```console
-notation sign --plain-http $IMAGE
+notation key list
 ```
 
-Use `notation list` to show the signatures for your container image.
+Use `notation certificate list` to confirm whether the certificate is stored in the trust store.
 
 ```console
-notation list --plain-http $IMAGE
+notation cert list
 ```
 
-Confirm there is one listed, for example:
+## Sign the container image
 
-```output
-$ notation list --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
-```
+Use `notation sign` to sign the container image with COSE signature format.
 
-## Verify the signature of an image
+```console
+notation sign --plain-http --signature-format cose $IMAGE
+```
 
-Use `notification verify` to verify any signatures on your container image.
+Use `notation list` to show the signatures associated with the container image.
 
 ```console
-notation verify --plain-http $IMAGE
+notation list --plain-http $IMAGE
 ```
 
-Confirm your container image fails validation. For example:
+Confirm there is one signature, for example:
 
 ```output
-$ notation verify --plain-http $IMAGE
-Error: trust certificate not specified
-2022/09/30 15:24:14 trust certificate not specified
+$ notation list --plain-http $IMAGE
+localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── application/vnd.cncf.notary.v2.signature
+    └── sha256:ba3a68a28648ba18c51a479145fca60d96b43dc96c6ab22f412c89ac56a9038b
 ```
 
-This verification fails because you have not trusted the public key for your container image.
+## Create a trust policy
 
-## Add a trusted public key
+In order to verify the container image, you need to configure the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. The trust policy is a JSON document with the name `trustpolicy.json` under `$HOME/.config/notation/`.
```

Comment:
Provide link to local filesystem document since $HOME doesn't work for everyone.  Also, might be a "cheat" to grab that path from the generated certificate with `notation cert ls`